### DISCUSSION TITLE
Error during HFS update in Windows

### DISCUSSION THREAD

**Discussion by Baggio824** [rejetto/hfs#545] [General]
Hi all! I'm not an expert in node.js :)
When I try to update HFS, an error appears during the update of HFS in Windows 11. How to fix it?

I will be grateful if you help!

![2024-04-01_114133](https://github.com/rejetto/hfs/assets/6585926/8e6e1da4-8e49-41e4-aba9-27ef129511d5)

**Comment by rejetto** [comment]
try `npx hfs@latest`

**Reply by Baggio824** [reply]
Thank you!
![2024-04-01_134223](https://github.com/rejetto/hfs/assets/6585926/501ee7ff-62ad-4db3-a000-226e4597684c)
[2024-04-01T10_41_56_529Z-debug-0.log](https://github.com/rejetto/hfs/files/14822682/2024-04-01T10_41_56_529Z-debug-0.log)

I executed the command, now the following error appears and this is in the logs:

**Comment by rejetto** [comment]
it seems you have to install git.
only now i'm understanding that to use npm or npx you need to first install git.

**Comment by rejetto** [comment]
this is not good, i don't want to ask people to install git, but i have an idea on how to solve this.
i will attempt a fix on next release.

**Comment by rejetto** [comment]
i've managed to solve this problem, so now you can `npx -y hfs@latest`

**Reply by PH5HP** [reply]
Hi,

What does this exactly? And how to install this?

Kind regards,
Henk - https://fileserver.ph5hp.nl

**Reply by rejetto** [reply]
this is just one of the installation methods described in the README
https://github.com/rejetto/hfs?tab=readme-ov-file#other-systems
